db/db_fingerprint.c

```diff
@@ -195,8 +198,6 @@ void add_fingerprint(struct sqlclntstate *clnt, sqlite3_stmt *stmt, const char *
         t->nNormSql = nNormSql;
         hash_add(gbl_fingerprint_hash, t);
 
-        char fp[FINGERPRINTSZ*2+1]; /* 16 ==> 33 */
```

Comment:
Maybe duplicate it in the place where we issue the warning?  Otherwise we're running this code for every query, though we only need it in one case.

Reply:
Fixed!